src/packages/covid/forms/sample-collection-form/1.0.json

```diff
@@ -350,6 +350,33 @@
                 }
               ]
             },
+            {
+              "label": "Reference Lab sample sent to",
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "text",
+                "concept": "161562AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "rows": "5",
+                "conceptMappings": [                  
+                  {
+                    "type": "CIEL",
+                    "value": "161562"
+                  }
+                ]
+              },
+              "id": "referenceLabSampleSentTo",
+              "behaviours": [
+                {
+                  "intent": "*",
+                  "required": "false",
+                  "unspecified": "true",
```

Comment:
@ODORA0  Could we remove the unspecified and required? The spec does not specify this.

Reply:
@alaboso Done